addons/project/static/src/js/tours/project.js

```diff
@@ -44,6 +44,7 @@ tour.register('project_tour', {
     run: "text Test",
 }, {
     trigger: ".o_kanban_project_tasks .o_column_quick_create .o_kanban_add",
+    content: Markup(_t('Let\'s create your first <b>column</b>.')),
```

Comment:
I think we should call them stages rather than columns
```suggestion
    content: Markup(_t('Let\'s create your first <b>stage</b>.')),
```